### PR TITLE
Improve project creation form copy

### PR DESCRIPTION
**Pull request by aeolianeth**
## What does this PR do and why?

This is the first iteration in a series of copy updates from [Notion](https://www.notion.so/juicebox/Design-Your-Project-Draft-4-b6fe073fdb0f44f8b6422ae08a567fa0).

In this PR, we focus on the `Funding` section of project creation. Key notes:

- [People don't read](https://www.nngroup.com/articles/how-users-read-on-the-web/). To make project creation easier, we need to reduce the amount of text needed.
- We can limit text by linking out to documentation when longer prose is needed.

## Screenshots or screen recordings

Screenshots below show each state of the form, before and after this PR.

| Before | After |
| --- | --- |
| Screen Shot 2021-12-18 at 6 24 55 pm | Screen Shot 2021-12-18 at 6 25 05 pm |
| Screen Shot 2021-12-18 at 6 14 56 pm | Screen Shot 2021-12-18 at 6 05 32 pm |
| Screen Shot 2021-12-18 at 6 15 21 pm | Screen Shot 2021-12-18 at 6 05 41 pm |
| Screen Shot 2021-12-18 at 6 15 52 pm | Screen Shot 2021-12-18 at 6 05 49 pm |
| Screen Shot 2021-12-18 at 6 16 13 pm | Screen Shot 2021-12-18 at 6 06 01 pm |
| Screen Shot 2021-12-18 at 6 17 48 pm | Screen Shot 2021-12-18 at 6 07 22 pm  |
| Screen Shot 2021-12-18 at 6 18 08 pm | Screen Shot 2021-12-18 at 6 07 01 pm |

## Acceptance checklist

- [x] I have evaluted the
      [Approval Guidelines](../../CONTRIBUTING.md#approval-guidelines) for this
      PR.

### CODE REVIEW

**Comment by aeolianeth** [comment]
Hey @peripheralist, @mejango, as Juicebox experts, would you both be able to take a look at this when you get a chance?

**Comment by mejango** [comment]
I like these a lot. The "Before" and "After" screenshots are really great for reviewing this type of work. Thanks, awesome work!

**Comment by mejango** [comment]
Also I appreciate how this is scoped to just a few sections.

File: src/components/Create/index.tsx [hunk 1]
```diff
@@ -454,13 +454,13 @@ export default function Create() {
...
             {
-              title: 'Appearance',
+              title: 'Project details',
               description: 'Project name, handle, links, and other details.',
               callback: () => setProjectFormModalVisible(true),
             },
             {
               title: 'Funding',
-              description: 'How your project will earn and manage funds.',
+              description: "Your project's funding cycle target and duration.",
```
Comment (aeolianeth) [inline src/components/Create/index.tsx:463]: Loosely held opinion. I think there needs to be mention of funding cycle, ideally.

**Comment by aeolianeth** [comment]
Hey @peripheralist, could you give this a final review? 🙇

File: src/components/Create/BudgetForm.tsx [hunk 2]
```diff
@@ -43,25 +44,47 @@ export default function BudgetForm({
...
         <div style={{ color: colors.text.secondary }}>
-          <h4>Target</h4>
+          <h4>Funding target</h4>
           <p>
-            No more than the target can be distributed from the project in a
-            single funding cycle. Whenever a new funding cycle starts, any
-            overflow automatically goes towards that cycle's target amount,
-            acting as a project's runway.
+            Set the amount of funds you'd like to raise each funding cycle. Any
+            funds raised within the funding target can be distributed by
```
Comment (peripheralist) [inline src/components/Create/BudgetForm.tsx:74]: "distributed by project" >> "distributed from your project"?

File: src/components/Create/BudgetForm.tsx [hunk 3]
```diff
@@ -101,11 +124,18 @@ export default function BudgetForm({
           />
         )}
 
-        {showFundingFields && (
+        {showFundingFields && target === '0' && (
```
Comment (peripheralist) [inline src/components/Create/BudgetForm.tsx:127]: this was originally conditional for when target == 0, but generally i've started leaning more towards showing things unconditionally. my assumption is that it's more helpful for folks to understand how the target works, even if they aren't setting their target to 0. wdyt @aeolianeth ?

File: src/components/Create/BudgetForm.tsx [hunk 4]
```diff
@@ -101,11 +124,18 @@ export default function BudgetForm({
...
-            can be distributed by the project, and the project's entire balance
-            will be considered overflow.
+            <span style={{ fontWeight: 600 }}>Target is 0.</span> The project's
+            entire balance will be considered overflow.{' '}
+            <a
+              href={helpPagePath('protocol/learn/topics/overflow')}
+              rel="noopener noreferrer"
+              target="_blank"
+            >
+              Learn more
```
Comment (peripheralist) [inline src/components/Create/BudgetForm.tsx:136]: love these new Learn More links 🙏

File: src/components/Create/index.tsx (hunk 1 above)
Comment (peripheralist) [reply src/components/Create/index.tsx:463]: I agree, a nice and consistent repetition of JB vocab seems like a good strategy

**Review by peripheralist — APPROVED** [review]
I'm really into these. Also appreciate that it's well-scoped, and love the new "Learn more" link pattern.

it's super refreshing to have new folks critiquing these communication strategies from a high level—even subtle copy changes can go a long way, and these changes feel good.

**Pull request merged**